.circleci/config.yml

```
@@ -91,11 +92,10 @@ jobs:
             gsutil -m rm -rf ${GCS_ROOT}/${CIRCLE_WORKFLOW_ID}
             gsutil -m rm -rf ${GCS_ROOT_URL}/${CIRCLE_WORKFLOW_ID}
 
- 
+
 # Define workflow here
 workflows:
   version: 2
   build_workflow:
     jobs:
       - pytest
```

It makes sense to run pre commit as part of CircleCI tests. Otherwise code can easily slip through that hasn't undergone the hooks. Something like this (assuming `lint` is a CircleCI job that runs the hooks):
```suggestion
    jobs:
      - lint
      - pytest:
          requires:
            lint
```

---

Added test `lint` as you suggested. It runs `pre-commit run --all`.